index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Cormorant">
+</head>
+<body>
+  <header>
+    <h1 class="title">M. Chau</h1>
+    <nav>
+      <a href=".">About</a>
+      <a href="pages/portfolio.html">Portfolio</a>
+      <a href="pages/contact.html">Contact</a>
+    </nav>
+  </header>
+  <section>
+    <h2>M. Chau is a software developer & writer & full-time dream machine.</h2>
```

Comment:
The HTML validator flagged this line because you used raw ampersands `&` here, instead of the `&amp;` escape. It ends up displaying just fine, though.